Providers/Profile.swift

```diff
@@ -502,21 +486,6 @@ public class BrowserProfile: Profile {
         return SQLiteLogins(db: self.loginsDB)
     }()
 
-    // This is currently only used within the dispatch_once block in loginsDB, so we don't
-    // have to worry about races giving us two keys. But if this were ever to be used
-    // elsewhere, it'd be unsafe, so we wrap this in a dispatch_once, too.
-    lazy private var loginsKey: String? = {
-        let key = "sqlcipher.key.logins.db"
```

Comment:
I was waiting for you or @farhanpatel to notice that this was completely screwed. 🤓 I think I commented about this on one of the migration PRs… be super careful! Read the entire migration diff, see what else might have gotten lost.